app/model/ProgressStatuses.scala

```diff
@@ -111,6 +111,8 @@ object ProgressStatuses {
   case object SIFT_ENTERED extends ProgressStatus(ApplicationStatus.SIFT)
   case object SIFT_READY extends ProgressStatus(ApplicationStatus.SIFT)
   case object SIFT_COMPLETED extends ProgressStatus(ApplicationStatus.SIFT)
+  case object FAILED_AT_SIFT extends ProgressStatus(ApplicationStatus.FAILED_AT_SIFT)
+  case object SDIP_FAILED_AT_SIFT extends ProgressStatus(ApplicationStatus.SIFT)
```

Comment:
I've seen there a couple more examples of adding scheme information to Progress status, but is it essential
here to have sdip specific progress status? Is that for reports?

Reply:
We need it for Sdip as it's not just an additional scheme, it's a whole application in itself.